router/bench-random.js

```diff
@@ -155,7 +155,7 @@ fragment EmployeeNameFragment on Employee {
     };
 
 
-    let operationName = randomString(10, 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789')
+    let operationName = randomString(10, 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz')
```

Comment:
No more numbers/no underscores so you don't have to have logic for valid operation names (e.g., begin with a number)?

Reply:
Yeah. I think `52^10 = 144555105949057020` possibilities should be good enough

Reply:
Much less about the number of possibilities and more about understanding the change